### ISSUE TITLE
"Failed to fetch dynamically imported module" when a space is present in file path during CT

### ISSUE THREAD

**Issue by Bobakanoosh** [cypress-io/cypress#18322]
### Current behavior

During component testing, if I run `cypress open-ct`, chrome opens fine, but once my component test tries to run, I get the following errors:

From Cypress:
  > Failed to fetch dynamically imported module: http://localhost:3000/__cypress/iframes/C:/Users/bobakanoosh/Documents/My%20Project/My_Project/src/components/Example.cy.js?import
  
From Cypress console
  > Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "". Strict MIME type checking is enforced for module scripts per HTML spec.
  
After hours of debugging and googling, I realized maybe my project path had some weirdness to it. I changed the following:
> C:/Users/bobakanoosh/Documents/My%20Project/My_Project/src/components/Example.cy.js?import
to
> C:/Users/bobakanoosh/Documents/MyProject/My_Project/src/components/Example.cy.js?import

And everything magically worked.

### Desired behavior

Spaces should be allowed in file paths.

### Test code to reproduce

```
import { mount } from "@cypress/vue";

import Example from "./Example.vue";

describe("<Example/>", () => {
	it("example", () => {
		const mounted = mount(Example);
		mounted.get("p").should("have.text", "Message");
	});
});
```

### Cypress Version

Cypress package version: 8.4.1 
Cypress binary version: 8.4.1 
Electron version: 13.2.0      
Bundled Node version:         14.16.0

**Comment by JessicaSachs** [comment]
I believe this was fixed as a part of https://github.com/cypress-io/cypress/pull/20575 and released in `@cypress/webpack-dev-server@1.8.3` can you give it a try?

**Comment by lmiller1990** [comment]
Fixed spaces in #20575
